src/RiotPls.DataDragon/Helpers/ValueTaskHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace RiotPls.DataDragon.Helpers
+{
+    internal static class ValueTaskHelper
```

Comment:
I still don't know if I should delete this class.

Reply:
What's wrong with it?

Reply:
I initially made it to make stuff shorter, but we end up with a mess that is just hard to maintain, also DataDragonClient.Internal already solves a lot of stuff I made this helper for, so it could be deleted and make small changes to the methods that still use it (like GetVersionsAsync and such)